### PR TITLE
Fred changes

### PR DESCRIPTION
**Pull request by palango**
## Description

Read claims from file during startup.

### CODE REVIEW

File: raiden/claim.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    TokenNetworkAddress,
+    TokenNetworkRegistryAddress,
+    TransactionHash,
+)
+from raiden_contracts.utils.type_aliases import ChainID, ChannelID, Signature, TokenAmount
+
+CLAIM_FILE_PATH = Path("./claims.json")
+DEFAULT_SETTLE_TIMEOUT = BlockTimeout(100)
+DEFAULT_REVEAL_TIMEOUT = BlockTimeout(50)
+TOKEN_ADDRESS = TokenAddress(to_canonical_address("0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"))
```
Comment (konradkonrad) [inline raiden/claim.py:50]: How do we make sure we keep track of such short-cut constants?

Comment (palango) [reply raiden/claim.py:50]: I'll create an issue.

My idea is that we shouldn't create token networks in the end, but for that we need to do some more work.

File: raiden/storage/serialization/types.py [hunk 2]
```diff
@@ -84,6 +84,7 @@
 
 MESSAGE_NAME_TO_QUALIFIED_NAME = {
     "AuthenticatedMessage": "raiden.messages.abstract.AuthenticatedMessage",
+    "Claim": "raiden.claim.Claim",
```
Comment (palango) [inline raiden/storage/serialization/types.py:87]: We should have a look at this again, maybe there's a nicer solution.

File: raiden/claim.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+                fee_schedule=FeeScheduleState(),
+                our_state=our_state,
+                partner_state=partner_state,
+                open_transaction=SuccessfulTransactionState(BlockNumber(0)),
+                close_transaction=None,
+                settle_transaction=None,
+            )
+
+            state_changes.append(
+                ContractReceiveChannelNew(
```
Comment (konradkonrad) [inline raiden/claim.py:169]: I would argue, that we only want to do this once. On the other hand, `raiden.transfer.token_network.handle_channelnew` seems to happily ignore duplicated `channelnew` events.

File: raiden/claim.py [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+                + self.owner
+            )
+        return ChannelID(int.from_bytes(bytes=hashed_id, byteorder="big"))
+
+
+def parse_claims_file() -> List[Claim]:
+    if not CLAIM_FILE_PATH.exists():
+        return []
+
+    # TODO: add error handling
```
Comment (palango) [inline raiden/claim.py:107]: TODO

File: raiden/claim.py [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+def claims_to_blockchain_events(
+    claims: List[Claim],
+    node_address: Address,
+    token_network_registry_address: TokenNetworkRegistryAddress,
+) -> List[StateChange]:
+    state_changes = create_new_token_network_event(
+        token_network_registry_address, claims[0].token_network_address
+    )
+
+    # TODO: check claim signature
```
Comment (palango) [inline raiden/claim.py:143]: TODO

**Pull request merged**